package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "chats",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "dev": "./node_modules/.bin/parcel ./static/index.html",
```

```suggestion
    "dev": "parcel ./static/index.html",
```

Лучше пользоваться интерфейсом, который предоставляет пакет. Пусть то бина может поменяться